### DISCUSSION TITLE
Cannot use Back Camera on GitHub Education input form

### DISCUSSION THREAD

**Discussion by Manju-S-6271** [community/community#113224] [GitHub Education] [answered]
### Hi everyone,

Hi, I am writing to you today because I am having difficulty using the Back Camera on my smartphone with the input form.

The information on my device is as follows:
```
Smartphone model: iPhone SE 3rd generation | NMYC3J/A
OS version: iOS 17.3.1 (not jailbroken or modified)
Browser: Google Chrome
```

**Comment by samus-aran** [comment, marked as answer]
Hi @Manju-S-6271 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I can only use my Front Camera to take a photo.](https://github.com/orgs/community/discussions/111352#use-front-camera)

> We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a phone to capture.
> 1. Please ensure that you have allowed camera access on the browser you're using to apply. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers).
> 2. If you can, print off your evidence and take a photo of this copy.
> 3. Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone.
> 4. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
> 5. Alternatively you may use another mobile device or computer.

**Reply by Manju-S-6271** [reply]
Thank you Mr. Samus.
I used DroidCam to try it out and it turned out rather well. (Either way, though, all my applications are rejected. I wonder if my certificate is bad...?)
Thank you very much for your solution.

**Reply by rakeshbala1998** [reply]
There is no use sharing this updated document. I've tested it on nearly all browsers with no success in switching to the back camera. This situation is unacceptable. Why can’t it just be uploaded? It seems clear that GitHub is intentionally allowing this bug to persist to avoid offering educational enrollment while promoting its educational support.

**Reply by erenuysaldev** [reply]
> There is no use sharing this updated document. I've tested it on nearly all browsers with no success in switching to the back camera. This situation is unacceptable. Why can’t it just be uploaded? It seems clear that GitHub is intentionally allowing this bug to persist to avoid offering educational enrollment while promoting its educational support.

same problem

**Comment by h1ac** [comment]
I used the opera browser and It let me choose the back camera! I got accepted :)
